src/backend/aspen/util/split.py

```diff
@@ -2,6 +2,8 @@
 from splitio.client.client import Client as SplitioClient
 from splitio.exceptions import TimeoutException
 
+from aspen.database.models import Group, User
```

Comment:
This is where we fix the split.io stuff. Everything else is fixing test url's and minor updates to test coverage.